global.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "1.0.45"
   },
   "sdk": {
-    "version": "3.1.301",
+    "version": "5.0.100",
```

Comment:
Will [this](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#prerequisites) need to be updated too?

Reply:
Great catch!